frappe/utils/bench_helper.py

```diff
@@ -50,14 +50,16 @@ def app_group(ctx, site=False, force=False, verbose=False, profile=False):
 		ctx.info_name = ''
 
 def get_sites(site_arg):
-	if site_arg and site_arg == 'all':
+	if site_arg == 'all':
 		return frappe.utils.get_sites()
-	else:
-		if site_arg:
-			return [site_arg]
-		if os.path.exists('currentsite.txt'):
-			with open('currentsite.txt') as f:
-				return [f.read().strip()]
+	elif site_arg:
+		return [site_arg]
+	elif os.path.exists('currentsite.txt'):
+		with open('currentsite.txt') as f:
+			site = f.read().strip()
+			if site:
+				return [site]
+	return []
```

Comment:
@gavindsouza @Thunderbottom These two aren't equivalent. Was this intended?

Reply:
running just `bench console` or any bench command without the site arg would open the currentsite if it existed (even before this was implemented), so I think it was an intended behavior all along

Reply:
It breaks `bench doctor`